src/components/rewards/AchievementCard.jsx

```
@@ -0,0 +1,14 @@
+import { Card, CardContent } from "@/components/ui/card"
+
+export const AchievementCard = ({ achievement, isUnlocked, data }) => (
```

ESLint

[error] 'achievement' is defined but never used.

(no-unused-vars)